Examples/Heisenberg1d/heisenberg1d.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 The NetKet Authors - All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import netket as nk
+import numpy as np
+import jax
+
+# 1D Lattice
+L = 20
+
+g = nk.graph.Chain(length=L, pbc=True)
+# Lattice translation operations
+
+# Hilbert space of spins on the graph
+hi = nk.hilbert.Spin(s=1 / 2, N=g.n_nodes, total_sz=0)
+
+# Ising spin hamiltonian
+ha = nk.operator.Heisenberg(hilbert=hi, graph=g)
+
+# RBM Spin Machine
+ma = nk.models.RBMSymm(
+    permutations=g.periodic_translations(),
```

Comment:
you don't like the argument ```automorphisms``` we used to have? Also, why ```periodic_translations()```?

Reply:
1. The C++ `RbmSpinSymm` of the old days used the term permutations so I stuck to this. Also, in principle you can pass any sequence of permutations of lattice sites to the machine, regardless of whether they are graph automorphisms for a specific system or not (tough if they are not, that's probably not a good idea - but possible), so I do find this more natural.
2. The periodic translations are a subset of all automorphisms of `g`. It seemed more natural for me here to just impose those symmetries physically - after all, that's the most common approach so far.

Reply:
We could pass `RBMSymm(permutations=g.automorphisms(), ...)` (which is equivalent to `RBMSymm(permutations=g, ...)` here) in the example as well so people can compare the energies...

Reply:
I agree with Damian on this. What we used to call automorphisms are not automorphisms, but rather translations... (maybe translations instead of periodic_translations?)